src/utils/config/service-helpers.js

```diff
@@ -520,6 +521,7 @@ export function cleanServiceGroups(groups) {
         }
         if (type === "truenas") {
           if (enablePools !== undefined) cleanedService.widget.enablePools = JSON.parse(enablePools);
+          cleanedService.widget.nasType = nasType ?? "scale";
```

Comment:
```suggestion
          if (nasType !== undefined) cleanedService.widget.nasType = nasType;
```

Sorry for the second change here but I'd like to keep the default value encapsulated to the widget itself, not in service-helpers.